substrate/frame/tx-pause/src/mock.rs

```diff
@@ -21,13 +21,17 @@
 
 use super::*;
 use crate as pallet_tx_pause;
-
-use frame_support::{
-	derive_impl, parameter_types,
-	traits::{ConstU64, Everything, InsideBoth, InstanceFilter},
+use frame::{
+	prelude::frame_system,
+	runtime::{
+		prelude::{
```

Comment:
preludes are meant to be imported as prelude::*. 

Please read the instructions of the tracking issue as to how to do this migration more carefully.

Reply:
@kianenigma There are cases where only one item is needed in the prelude